site/static/main.ts

```diff
@@ -29,8 +29,10 @@ function renderExample($target: Selection<any, any, any, any>, text: string) {
 
   const vis = $target.append('div').attr('class', 'example-vis');
 
+  // Decrease visual noise by removing $schema and description from code examples.
+  const textClean = text.replace(/(\s)+\"(\$schema|description)\": \".*?\",/g, '');
```

Comment:
Yup, that's right. `$schema` and `description` properties remain in the original example specifications. My intention is to just remove them when displaying them on documentation pages to reduce the clutter.